src/components/transport_manager/src/usb/libusb/usb_connection.cc

```diff
@@ -205,7 +205,15 @@ bool UsbConnection::PostOutTransfer() {
 }
 
 void UsbConnection::OnOutTransfer(libusb_transfer* transfer) {
-  LOG4CXX_TRACE(logger_, "enter with  Libusb_transfer*: " << transfer);
+  LOG4CXX_AUTO_TRACE(logger_);
+  LOG4CXX_DEBUG(logger_,"enter with Libusb_transfer*: " << transfer);
```

Comment:
@Ypostolov this is at least trace. But I suppose that transfer pointer is redundant info in logs

Reply:
Fixed in 3c539a8a53cfd07be342e9a557446b71d8247c5a